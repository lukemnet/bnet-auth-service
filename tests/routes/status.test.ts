import server from '../../src/index';

describe('/status', () => {
  beforeAll(() => {
    server.registerPlugins();
  });

  it("GET returns 200", async () => {
    const response = await server.instance.inject({ method: "GET", url: "/status" });
    expect(response.statusCode).toEqual(200);
  });

  it("Response matches snapshot", async () => {
    const response = await server.instance.inject({ method: "GET", url: "/status" });
    const payload = JSON.parse(response.payload);
    expect(payload).toMatchSnapshot({ status: 200, message: 'ok' });
  });
});