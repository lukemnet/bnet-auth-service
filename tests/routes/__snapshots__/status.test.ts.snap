// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/status Response matches snapshot 1`] = `
Object {
  "message": "ok",
  "status": 200,
}
`;
